package.json

```diff
@@ -45,6 +46,7 @@
     "jest": "^24.8.0",
     "mocha": "^5.2.0",
     "monocle-ts": "^2.0.0",
+    "newtype-ts": "^0.3.2",
```

Comment:
travis is complaining, you should add this with `npm i newtype-ts -D` I guess

Reply:
Done